src/components/SarcophagusCreate/ArchaeologistSelect/ArchaeologistTableRow.tsx

```diff
@@ -44,7 +44,7 @@ const ArchaeolgistTableRow = ({
   const isArchaeologistMaxResValid = isMaxResurrectionTimeValid(archaeologist.maximumResurrectionTime.toNumber(), Number(values.resurrectionTime))
 
   const isDisabled = !isBountyLess || !isDiggingFeeLess || !isArchaeologistMaxResValid;
-  // checks freebond is greater
+  // delete?
   const isFreeBondGreater = archaeologist.freeBond.gte(ethers.utils.parseEther(archTotal));
```

Comment:
wasn't sure if we still want filter at row level now that it's moved up to the comp that manages the list state.

Reply:
yeah I agree, good call. let's delete this and the conditional blocking the render on line 65